packages/configtest/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@webpack-cli/configtest",
+  "version": "1.0.0",
+  "description": "Test your webpack configuration against validation errors.",
```

Comment:
```suggestion
  "description": "Tests webpack configuration against validation errors.",
```

Reply:
I'm not sure `Tests` will be suitable here. I replaced `your` with `any`. 
`Test any webpack configuration against validation errors.`

Reply:
It should be `Tests`

Reply:
fixed